project/webapp/templates/thread.html

```diff
@@ -6,6 +6,24 @@
     {% include "base/less_headers/thread_less.html" %}
 </head>
 
+<script>
+$(document).ready(function() {
+  $('#csv-export').click(function() {
+    $.ajax({
+      type: 'POST',
+      url: 'http://127.0.0.1:8000/civi2csv',
```

Comment:
We have to be careful about tying code too closely to our development environment. `localhost:8000` will work for you when working on your development computer, but will break when the site is deployed. So, change this URL to a relative path, rather than hard-coding it to the development environment.